client/src/rpc_request.rs

```diff
@@ -97,6 +97,11 @@ pub struct RpcVoteAccountInfo {
     /// Whether this account is staked for the current epoch
     pub epoch_vote_account: bool,
 
+    /// History of how many credits earned by the end of each epoch
+    ///   each tuple is (Epoch, credits, prev_credits)
+    #[serde(default)]
```

Comment:
Added this for backwards compatibility, do we need it?

Reply:
Backwards compatibility with what exactly?

Reply:
When using the latest cli client against a rpc endpoint without this value present, the request fails.

Reply:
Weird, do you know why?

Reply:
Maybe I didn't word that clearly...

With `#[serde(default)]`
```
cargo run -p solana-cli -- show-validators          
RPC Endpoint: http://beta.testnet.solana.com:8899
Active Stake: 0.001696972 SOL

  Identity Pubkey                               Vote Account Pubkey                           Commission  Last Vote  Root Block   Uptime  Active Stake
  va11wrZ2pD668e2dKXohuXiyALPxfVQjjH7zzpePavQ   7aWpg8sXjZVDLAxTohdjmmweW2RxQYWBFeBBRVLbUQN8         50%      53978       53947        -  0.000424243 SOL (25.00%)
  va13en4eUarJtf8mbhFF386nvQh12g6ESkjoR7Ji8hm   EU9SK7xG5YLpX2amTfaBuQpBFqCxvhMqxbCwbjwNb2A5         50%      53978       53947        -  0.000424243 SOL (25.00%)
  va12u4o9DipLEB2z4fuoHszroq1U9NcAB9aooFDPJSf   DTUGtGtHAdETxaUkNoDr5i9a1RgsFP9a9DoPY7cLQZpd         50%      53978       53947        -  0.000424243 SOL (25.00%)
  boot1Z6jb15CLqpaMTn2CxktktwZpRAVAgHZEW6SxQ7   3YTyUqUoZptsQHxwjxT5Yk7SBP5iL6tsWgZpn9ez4gFM          0%      53978       53947        -  0.000424243 SOL (25.00%)
```

Without `#[serde(default)]`
```
cargo run -p solana-cli -- show-validators
RPC Endpoint: http://beta.testnet.solana.com:8899
Error: Custom { kind: Other, error: "GetVoteAccounts parse failure: missing field `epochCredits`" }
```

Reply:
Oh I see, you're mixing master and v0.21.  I don't think we need this.  Once this PR lands on v0.21 this situation will resolve itself.